src/hypothesis/strategies.py

```diff
@@ -174,16 +174,18 @@ def one_of(*args):
         except TypeError:
             pass
 
+    strategies = []
```

Comment:
Does it? I still get reasonable-looking repr’s this way, and if I add `@defines_strategy`, something funky happens and the branches no longer get flattened.